wasm2c/wasm-rt-os-win.c

```diff
@@ -0,0 +1,256 @@
+// Based on
+// https://web.archive.org/web/20191012035921/http://nadeausoftware.com/articles/2012/01/c_c_tip_how_use_compiler_predefined_macros_detect_operating_system#BSD
+// Check for windows (non cygwin) environment
+#if defined(_WIN32)
+
+#include "wasm-rt-os.h"
+#include "wasm-rt.h"
+
+#include <errno.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <windows.h>
+
+#ifdef VERBOSE_LOGGING
+#define VERBOSE_LOG(...) \
+  { printf(__VA_ARGS__); }
+#else
+#define VERBOSE_LOG(...)
+#endif
+
+#define DONT_USE_VIRTUAL_ALLOC2
```

Comment:
Seems odd to have this always defined. Why not remove the other option below then?